cabal.project

```diff
@@ -51,35 +51,21 @@ constraints: these -assoc
 --
```

Comment:
I think this last NOTE can be removed

Reply:
Done - i separated the notes out to their constituent files.